sdk/identity/identity/TROUBLESHOOTING.md

```diff
@@ -12,6 +12,7 @@ This troubleshooting guide covers the following areas of the Azure Identity clie
   - [AuthenticationRequiredError](#authenticationrequirederror)
   - [CredentialUnavailableError](#credentialunavailableerror)
   - [AuthenticationError](#authenticationerror)
+  - [AggregateAuthenticationError](#aggregateauthenticationerror)
```

Comment:
It's best to alphabetize this list. I'd recommend reordering the "Handle Azure Identity errors" sub-sections accordingly.